presto-main/src/main/java/com/facebook/presto/operator/TaskContext.java

```diff
@@ -161,6 +171,8 @@ public void start()
         DateTime now = DateTime.now();
         executionStartTime.compareAndSet(null, now);
         startNanos.compareAndSet(0, System.nanoTime());
+        startFullGcCount.compareAndSet(0, gcMonitor.getMajorGcCount());
+        startFullGcTimeNanos.compareAndSet(0, gcMonitor.getMajorGcTime().roundTo(NANOSECONDS));
```

Comment:
0 is not a reasonable initial value for these two fields. Use -1 instead. I would imagine that these two values will stay at 0 for a long time after VM start.